lib/Passes/RemoveCallIntrinsics.cpp

```diff
@@ -43,23 +38,45 @@ RemoveCallIntrinsics::runOnIntrinsic(llvm::CallInst *remillFunctionCall,
       ra.references_global_value ||  // Related to a global var/func.
       ra.references_program_counter) {  // Related to `__anvill_pc`.
 
-    // TODO(Ian): ignoring callsite decls for now
-    auto fdecl = spec.FunctionAt(ra.u.address);
+    std::optional<CallableDecl> maybe_decl;
```

Comment:
Shouldnt this be initialized to spec.FunctionAt folded address ie. the default is we just call the address and dont have to rely on the override if one doesnt exist?